google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/GeneratedCreateTableStatementsTests.java

```diff
@@ -245,4 +248,19 @@ public void testCreateUniqueIndexes_oneToMany() throws SQLException {
         "RUN BATCH"
     );
   }
+
+  @Test
+  public void testRunBatchFails() {
+    this.connection
+        .getStatementResultSetHandler()
+        .prepareThrowsSQLException("RUN BATCH", new SQLException("test exception"));
+    Metadata metadata =
+        new MetadataSources(this.registry).addAnnotatedClass(Account.class).buildMetadata();
+
+    // Without the custom DdlTransactionIsolater that is returned by SpannerSchemaManagementTool,
+    // the SQLException would just be silently ignored by Hibernate.
+    SpannerException spannerException =
+        assertThrows(SpannerException.class, () -> metadata.buildSessionFactory().openSession());
```

Comment:
Please separate the chained method so that only one invocation should throw an exception.

Reply:
Done.